Plan/common/src/main/java/com/djrapitops/plan/settings/locale/Locale.java

```diff
@@ -36,6 +36,8 @@
  */
 public class Locale extends HashMap<Lang, Message> {
 
+    public static final Pattern COMPILE = Pattern.compile("(<script id=[\"|'].*[\"|']>[\\s\\S]*?</script>|<script>[\\s\\S]*?</script>|<script src=[\"|'].*[\"|']></script>|<link [\\s\\S]*?>)");
```

Comment:
This probably needs to be private and named something that explains what it does.
This regex finds script and link tags.

Reply:
The security issue is not resolved as it is related to the backtracking the regex performs.